packages/tailwindcss/src/index.ts

```diff
@@ -452,6 +453,23 @@ async function parseCss(
     firstThemeRule.nodes = nodes
   }
 
+  // Replace the `@tailwind utilities` node with a conext since it prints
+  // children directly.
+  if (utilitiesNode) {
+    let node = utilitiesNode as AstNode as Context
+    node.kind = 'context'
+    node.context = {}
+
+    // Remove additional `@tailwind utilities` nodes
+    walk(ast, (node, { replaceWith }) => {
+      if (node.kind !== 'at-rule') return
+      if (node.name !== '@tailwind') return
+      if (!node.params.startsWith('utilities')) return
+
+      replaceWith([])
+    })
```

Comment:
Can you think of any case where there are two `@tailwind utilities` nodes in the CSS file that is not "you screwed up your imports"? Wondering if we should just error on this case instead (since we already have a walk that will find `@tailwind utilities`.)

Reply:
We could. There's an explicit test for this though which is the only reason I even added this bit.

Reply:
Ah, makes sense. Yeah then maybe let's keep this behavior but I think we could still remove this walk and remove the node as part of the walk where we find the `utilitiesNode`:

```diff
     // Find `@tailwind utilities` so that we can later replace it with the
     // actual generated utility class CSS.
     if (
-      utilitiesNode === null &&
       node.name === '@tailwind' &&
       (node.params === 'utilities' || node.params.startsWith('utilities'))
     ) {
+      if (utilitiesNode !== null) {
+        replaceWith([])
+        return
       }
       let params = segment(node.params, ' ')
       for (let param of params) {
         if (param.startsWith('source(')) {
           let path = param.slice(7, -1)
           // Keyword: `source(none)`
           if (path === 'none') {
             root = path
             continue
           }
           // Explicit path: `source('…')`
           if (
             (path[0] === '"' && path[path.length - 1] !== '"') ||
             (path[0] === "'" && path[path.length - 1] !== "'") ||
             (path[0] !== "'" && path[0] !== '"')
           ) {
             throw new Error('`source(…)` paths must be quoted.')
           }
           root = {
             base: context.sourceBase ?? context.base,
             pattern: path.slice(1, -1),
           }
         }
       }
       utilitiesNode = node
     }
```

Reply:
ah ya good idea

Reply:
Updated